exampleCourse/questions/element/details/server.py

```diff
@@ -0,0 +1,10 @@
+import pandas as pd
+import prairielearn as pl
+
+
+def generate(data: pl.QuestionData) -> None:
+    # Generate a DataFrame with some sample data
+    df = pd.read_csv("breast-cancer-train.csv", header=None)
```

Comment:
Is there a more reasonable example to use here? The table offers no context on what it is or what the different columns are.

Reply:
I had looked for a large table under `element/dataframe` but didn't see one with headers that was large in size. Should I add that example to both that element and this one?

Reply:
Yeah, let's do that. Even if we can find the source for this data and add relevant headers, that'd be enough. (I don't know how we'd do that 😅)

Reply:
I threw it at `o3` and it identified the source as https://archive.ics.uci.edu/dataset/17/breast%2Bcancer%2Bwisconsin%2Bdiagnostic.

Reply:
Changed in [f81b596](https://github.com/PrairieLearn/PrairieLearn/pull/11929/commits/f81b59665f57416b3f4efda0af559efa039fcf2c). The `pl-dataframe` didn't need it added.